tests/dsl/src/test/java/org/mule/test/extension/dsl/ConfigurationBasedElementModelFactoryTestCase.java

```diff
@@ -29,27 +35,66 @@ public void initApp() throws Exception {
   }
 
   @Test
-  public void resolveSimpleConfigWithFlatConnection() throws Exception {
+  public void defaultValueResolution() throws Exception {
```

Comment:
test for "defaultValue from element", fixes => https://www.mulesoft.org/jira/browse/MULE-11537